doc/conf.py

```diff
@@ -141,7 +141,8 @@
     'sklearn': ('https://scikit-learn.org/stable', None),
     'numba': ('https://numba.readthedocs.io/en/latest', None),
     'joblib': ('https://joblib.readthedocs.io/en/latest', None),
-    'nibabel': ('https://nipy.org/nibabel', None),
+    'nibabel': ('https://nipy.org/nibabel',
+                'https://nipy.org/nibabel/objects.inv'),
```

Comment:
I don't think this was actually the problem; the problem was that the entire nipy.org site was down for a couple days. Can you try reverting this change (but keeping the changelog fix) and see if the CIs are happy that way?

Reply:
It looks like it's back up, will do.